packages/translate/src/index.js

```diff
@@ -305,6 +312,9 @@ Translate.prototype.getLanguages = function(target, callback) {
  *     is written in.
  * @param {string} options.to - The ISO 639-1 language code to translate the
  *     input to.
+ * @param {string} options.model - **Note:** Users must be whitelisted to use
```

Comment:
Done - originally I had not only because I thought `to` and `from` made more sense being grouped together.